Makefile.pre.in

```diff
@@ -1807,6 +1830,23 @@ altbininstall: $(BUILDPYTHON) @FRAMEWORKPYTHONW@
 			-output $(DESTDIR)$(BINDIR)/python$(VERSION)-intel64$(EXE) \
 			$(DESTDIR)$(BINDIR)/python$(VERSION)$(EXE); \
 	fi
+	# Install MacOS debug information (if available)
```

Comment:
It should be `macOS`, not `MacOS`, in all references here.

Reply:
This is inconsistent across all files. There is MacOS and macOS. I went for the most common one.

Reply:
Sorry to be pendantic about this but `MacOS` is incorrect. That refers to the historic classic Mac operating system that was retired 20 years ago. The [current Mac operating system](https://en.wikipedia.org/wiki/Macintosh_operating_systems) that Apple (and we) support has had three names over its 20-year life span: `Mac OS X`, then `OS X`, and since 2016 `macOS`.  We still have inconsistent references across the source code that we should eventually clean up but we should ensure that new code uses the correct form. I've taken the liberty of pushing a commit to the PR to use `macOS`.